### ISSUE TITLE
Pylint should not complain of django constants

### ISSUE THREAD

**Issue by lfrodrigues** [landscapeio/pylint-django#41]
in urls.py I get:
    pylint: invalid-name / Invalid constant name "urlpatterns"

When using ```register = template.Library()``` for a template tag I get:
    pylint: invalid-name / Invalid constant name "register"

I know I can add comment in code to ignore this but it would be better if support would be built in pylint.

**Comment by carlio** [comment]
Good point!

If you're interested in making a pull request, the basic idea is in this file:
https://github.com/landscapeio/pylint-django/blob/master/pylint_django/augmentations/__init__.py#L290 - choose a message and selectively turn it off given the context.

Otherwise I'll do it myself but I probably don't have much time for a few weeks.

**Comment by carlio** [comment]
Actually, this is already present: https://github.com/landscapeio/pylint-django/blob/master/pylint_django/augmentations/__init__.py#L246

(insert my confused face)

**Comment by lfrodrigues** [comment]
You're right, prospector wasn't detecting Django and not loading pylint-django...

**Comment by bittner** [comment]
Interesting. I'm using pylint with pylint-django, and pylint still complains about the usual lower-case constants used in Django (such as `urlpatterns`). Despite commit b5392e4.

I'm using Python 3.5.2, Django 1.10.4, pylint 1.6.4, astroid 1.4.9 and pylint-django 0.7.2. PyLint is run via tox 2.5.0. PyLint shows the pylint-django plugin loaded when running `pylint --help` (`tox -e pylint -- --help`, really).

What's wrong?

**Comment by sergafts** [comment]
I am having the same issue. I can see that pylint-django works, since I don't see warnings about `ForeignKey`s having no `objects` etc. However I get the same errors about `urlpatterns`. Any clues?

**Comment by thedarklord1** [comment]
+1
